teacher.rb

```diff
@@ -0,0 +1,11 @@
+require './person'
+class Teacher < Person
+  def initialize(specialization, name, parent_permission: 'true')
+    super(age, name, parent_permission)
```

Comment:
- [ ] Kindly make sure that when passing in the parameters to the `super` class, the order of the parameters matches with the super constructor. Currently, the super constructor accepts parameters in the following order:
https://github.com/altontonn/School-Library/blob/bd96b9a4fd7b1b3e7958f5c111a3b1e228a5e966/person.rb#L2

- [ ] Please add the `age` to the constructor, as you're passing it to the super constructor.